cypress/fixtures/returns-requirements.json

```
@@ -0,0 +1,238 @@
+      "firstName": "John",
+      "lastName": "Testerson",
+      "middleInitials": "J",
+      "contactType": "person",
+      "dataSource": "nald"
+    }
+  ],
+  "billingAccounts": [
+    {
+      "id": "16cb50a5-e3e6-41f4-a42b-9dad6a69fc0c",
+      "accountNumber": "A99999991A",
+      "companyId": "e8abdbb4-aeea-47d4-91b2-97bf82bc2778"
+    }
+  ],
+  "billingAccountAddresses": [
+    {
+      "id": "ab9a24ec-80b8-40dc-82f8-7df3e885245a",
+      "billingAccountId": "16cb50a5-e3e6-41f4-a42b-9dad6a69fc0c",
+      "addressId": "62549cdb-073f-4d5c-a2a1-c47b0b910010",
+      "startDate": "2022-04-01"
+    }
+  ],
+  "licenceDocuments": [
+    {
+      "id": "1a274f3e-f891-43dd-8c25-8afac4e760ac",
+      "licenceRef": "AT/TEST/01",
+      "startDate": "2022-04-01"
+    }
+  ],
+  "licenceDocumentRoles": [
+    {
+      "licenceDocumentId": "1a274f3e-f891-43dd-8c25-8afac4e760ac",
+      "licenceRoleId": {
+        "schema": "public",
+        "table": "licenceRoles",
+        "lookup": "name",
+        "value": "licenceHolder",
+        "select": "id"
+      },
+      "startDate": "2022-04-01",
+      "companyId": "e8abdbb4-aeea-47d4-91b2-97bf82bc2778",
+      "addressId": "62549cdb-073f-4d5c-a2a1-c47b0b910010",
+      "contactId": "6e05db31-39cd-4bb0-83a0-0d985037ad8f"
+    }
+  ],
+  "permitLicences": [
+    {
+      "licenceRef": "AT/TEST/01",
+      "startDate": "2022-04-01",
+      "metadata": {
+        "source": "acceptance-test-setup"
+      }
+    }
+  ],
+  "licenceDocumentHeaders": [
+    {
+      "id": "282b226e-c47b-4dcc-bbb0-94648fb6b242",
+      "regimeEntityId": {
+        "schema": "crm",
+        "table": "entity",
+        "lookup": "entityType",
+        "value": "regime",
+        "select": "entityId"
+      },
+      "licenceRef": "AT/TEST/01",
+      "naldId": {
+        "schema": "public",
+        "table": "permitLicences",
+        "lookup": "licenceRef",
+        "value": "AT/TEST/01",
+        "select": "id"
+      },
+      "metadata": {
+        "Name": "Big Farm 01",
+        "dataType": "acceptance-test-setup",
+        "IsCurrent": true
+      }
+    }
+  ],
+  "licences": [
+    {
+      "id": "f8702a6a-f61d-4b0a-9af3-9a53768ee516",
+      "licenceRef": "AT/TEST/01",
+      "regionId": "d0a4123d-1e19-480d-9dd4-f70f3387c4b9",
+      "regions": {
+        "historicalAreaCode": "SAAR",
+        "regionalChargeArea": "Southern"
+      },
+      "startDate": "2022-04-01",
+      "waterUndertaker": true
+    }
+  ],
+  "licenceVersions": [
+    {
+      "id": "b846afb6-9334-42c4-8da3-1d559a6e3ea7",
+      "licenceId": "f8702a6a-f61d-4b0a-9af3-9a53768ee516",
+      "issue": 1,
+      "increment": 0,
+      "status": "current",
+      "startDate": "2022-04-01",
+      "externalId": "6:1234:1:0"
+    },
+    {
+      "id": "2f9362e8-4fdc-4018-880a-31bba8d9b23b",
+      "licenceId": "f8702a6a-f61d-4b0a-9af3-9a53768ee516",
+      "issue": 1,
+      "increment": 0,
+      "status": "current",
+      "startDate": "2022-04-01",
```

Same applies to the start date. I'd set it some time after version 1.

```suggestion
      "startDate": "2023-06-12",
```